tfproviderlint panics on interface conversion

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### tfproviderlint and terraform-plugin-sdk Version

```console
$ tfproviderlint -version
tfproviderlint v0.18.0

$ go mod graph | cut -d' ' -f 1 | grep terraform-plugin-sdk | uniq
github.com/hashicorp/terraform-plugin-sdk@v1.0.0
```

### Expected Behavior

I expect tfproviderlint to give me useful tips.

### Actual Behavior

```
$ tfproviderlint ./...
panic: interface conversion: types.Object is *types.Var, not *types.PkgName

goroutine 4479 [running]:
github.com/bflad/tfproviderlint/helper/astutils.IsModulePackageFunc(0x1450300, 0xc001a73d00, 0xc001a76a50, 0x13d5f63, 0x29, 0x13cb8b0, 0x11, 0x13c7ded, 0x7, 0x203000)
	/home/runner/work/tfproviderlint/tfproviderlint/helper/astutils/package.go:41 +0x2bd
github.com/bflad/tfproviderlint/helper/terraformtype/helper/validation.IsFunc(0x1450300, 0xc001a73d00, 0xc001a76a50, 0x13c7ded, 0x7, 0x100000000000)
	/home/runner/work/tfproviderlint/tfproviderlint/helper/terraformtype/helper/validation/package.go:21 +0x84
github.com/bflad/tfproviderlint/helper/analysisutils.SelectorExprRunner.func1.1(0x144bb00, 0xc001a73d00)
	/home/runner/work/tfproviderlint/tfproviderlint/helper/analysisutils/runners.go:470 +0x7c
golang.org/x/tools/go/ast/inspector.(*Inspector).Preorder(0xc001e94f00, 0xc002f51af0, 0x1, 0x1, 0xc002f51b18)
	/home/runner/work/tfproviderlint/tfproviderlint/vendor/golang.org/x/tools/go/ast/inspector/inspector.go:77 +0x9f
github.com/bflad/tfproviderlint/helper/analysisutils.SelectorExprRunner.func1(0xc00308a900, 0xc00308a900, 0x0, 0x0, 0x0)
	/home/runner/work/tfproviderlint/tfproviderlint/helper/analysisutils/runners.go:467 +0x149
golang.org/x/tools/go/analysis/internal/checker.(*action).execOnce(0xc001e68460)
	/home/runner/work/tfproviderlint/tfproviderlint/vendor/golang.org/x/tools/go/analysis/internal/checker/checker.go:690 +0x83a
sync.(*Once).doSlow(0xc001e68460, 0xc0004f1790)
	/opt/hostedtoolcache/go/1.14.7/x64/src/sync/once.go:66 +0xec
sync.(*Once).Do(...)
	/opt/hostedtoolcache/go/1.14.7/x64/src/sync/once.go:57
golang.org/x/tools/go/analysis/internal/checker.(*action).exec(0xc001e68460)
	/home/runner/work/tfproviderlint/tfproviderlint/vendor/golang.org/x/tools/go/analysis/internal/checker/checker.go:579 +0x60
golang.org/x/tools/go/analysis/internal/checker.execAll.func1(0xc001e68460)
	/home/runner/work/tfproviderlint/tfproviderlint/vendor/golang.org/x/tools/go/analysis/internal/checker/checker.go:567 +0x34
created by golang.org/x/tools/go/analysis/internal/checker.execAll
	/home/runner/work/tfproviderlint/tfproviderlint/vendor/golang.org/x/tools/go/analysis/internal/checker/checker.go:573 +0x119

```

--- a/helper/astutils/package.go
+++ b/helper/astutils/package.go
@@ -38,7 +38,10 @@ func IsModulePackageFunc(e ast.Expr, info *types.Info, module string, packageSuf
 
 		switch x := e.X.(type) {
 		case *ast.Ident:
-			return isModulePackagePath(module, packageSuffix, info.ObjectOf(x).(*types.PkgName).Imported().Path())
+			switch xObj := info.ObjectOf(x).(type) {
+			case *types.PkgName:
+				return isModulePackagePath(module, packageSuffix, xObj.Imported().Path())
+			}
 		}
 	case *ast.StarExpr:
 		return IsModulePackageFunc(e.X, info, module, packageSuffix, funcName)
@@ -60,7 +63,10 @@ func IsModulePackageFunctionFieldListType(e ast.Expr, info *types.Info, module s
 
 		switch x := e.X.(type) {
 		case *ast.Ident:
-			return isModulePackagePath(module, packageSuffix, info.ObjectOf(x).(*types.PkgName).Imported().Path())
+			switch xObj := info.ObjectOf(x).(type) {
+			case *types.PkgName:
+				return isModulePackagePath(module, packageSuffix, xObj.Imported().Path())
+			}
 		}
 	case *ast.StarExpr:
 		return IsModulePackageFunctionFieldListType(e.X, info, module, packageSuffix, typeName)
@@ -126,7 +132,10 @@ func IsPackageFunc(e ast.Expr, info *types.Info, packageSuffix string, funcName 
 
 		switch x := e.X.(type) {
 		case *ast.Ident:
-			return strings.HasSuffix(info.ObjectOf(x).(*types.PkgName).Imported().Path(), packageSuffix)
+			switch xObj := info.ObjectOf(x).(type) {
+			case *types.PkgName:
+				return strings.HasSuffix(xObj.Imported().Path(), packageSuffix)
+			}
 		}
 	case *ast.StarExpr:
 		return IsPackageFunc(e.X, info, packageSuffix, funcName)
@@ -148,7 +157,10 @@ func IsPackageFunctionFieldListType(e ast.Expr, info *types.Info, packageSuffix 
 
 		switch x := e.X.(type) {
 		case *ast.Ident:
-			return strings.HasSuffix(info.ObjectOf(x).(*types.PkgName).Imported().Path(), packageSuffix)
+			switch xObj := info.ObjectOf(x).(type) {
+			case *types.PkgName:
+				return strings.HasSuffix(xObj.Imported().Path(), packageSuffix)
+			}
 		}
 	case *ast.StarExpr:
 		return IsPackageFunctionFieldListType(e.X, info, packageSuffix, typeName)
@@ -215,7 +227,10 @@ func IsStdlibPackageFunc(e ast.Expr, info *types.Info, packagePath string, funcN
 
 		switch x := e.X.(type) {
 		case *ast.Ident:
-			return info.ObjectOf(x).(*types.PkgName).Imported().Path() == packagePath
+			switch xObj := info.ObjectOf(x).(type) {
+			case *types.PkgName:
+				return xObj.Imported().Path() == packagePath
+			}
 		}
 	case *ast.StarExpr:
 		return IsStdlibPackageFunc(e.X, info, packagePath, funcName)
@@ -237,7 +252,10 @@ func IsStdlibPackageFunctionFieldListType(e ast.Expr, info *types.Info, packageP
 
 		switch x := e.X.(type) {
 		case *ast.Ident:
-			return info.ObjectOf(x).(*types.PkgName).Imported().Path() == packagePath
+			switch xObj := info.ObjectOf(x).(type) {
+			case *types.PkgName:
+				return xObj.Imported().Path() == packagePath
+			}
 		}
 	case *ast.StarExpr:
 		return IsStdlibPackageFunctionFieldListType(e.X, info, packagePath, typeName)

